src/main/java/com/example/Restaurant_Management_System_REST_API/model/entity/MenuRecord.java

```diff
@@ -0,0 +1,22 @@
+package com.example.Restaurant_Management_System_REST_API.model.entity;
+
+import com.example.Restaurant_Management_System_REST_API.model.Category;
+import com.example.Restaurant_Management_System_REST_API.model.CatalogItem;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+public class MenuRecord extends CatalogItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    private List<String> ingredients;
```

Comment:
I think there might be a problem with this type. Is List enough here? Convince me! :D

Reply:
I haven't thought about this!  But when I think I think it is good idea to change it to Set! I will not have duplicate elements, I don't need an insertion order, I will not have null values!
Then I change to Set. is that fair enough?

Reply:
Not sure if any of those what you mentioned is a problem here :) Stick with list or change to Set, whichever you prefer then. I'll come back to this conversation in future PR.